docs/custom_head.html

```diff
@@ -0,0 +1,17 @@
+<link rel="icon" href="https://realm.io/img/favicon.ico">
+    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="https://realm.io/img/favicon-57x57.png" />
```

Comment:
The indentation looks inconsistent here. Most of these `link` elements have an unnecessary trailing `/`.